### PR TITLE
Coal Liquefaction

### PR DESCRIPTION
**Pull request by HonkingGoose**
- Changed Coal Liquefaction ratios.
- For Accurate Liquefaction + cracking: removed plastic bar production.
- Changed links from Reddit to wiki.

### CODE REVIEW

**Comment by HonkingGoose** [comment]
@deniszholob My commit 5a6ba3dd7e925685ce7a142409704a08f01455d2 breaks the icons next to the numbers. So I'm doing something wrong here...

File: src/assets/data/oil-refining.json [hunk 1]
```diff
@@ -67,34 +67,30 @@
...
             {
                 "iconId": "Heavy_oil_cracking",
-                "count": 3
+                "count": 39
             },
             {
                 "iconId": "Light_oil_cracking",
-                "count": 9
+                "count": 55
             },
```
Comment (deniszholob) [inline src/assets/data/oil-refining.json]: extra comma i think
data is not loading, and there is a console error
https://deploy-preview-118--factoriocheatsheet.netlify.com/

Comment (HonkingGoose) [reply src/assets/data/oil-refining.json]: I've removed the comma at the end of the section.

**Comment by HonkingGoose** [comment]
There's still an issue, there's a floating 75 here: 

screenshot from 2019-02-27 03-16-47

`<div _ngcontent-c9="" class="factorio-icon-text">75</div>`

**Comment by deniszholob** [comment]
Thats 75 is for blue science, the icon name changed in wiki, needs updating, ill create another issue for that.
the oil ratio is also updated to 10-1-7 from 8-1-7

**Comment by HonkingGoose** [comment]
The oil section on the live page still has:
>If you want to convert all refinery products to petroleum: for every 25 refineries you have running Advanced Oil Processing, you need 3 chemical plants cracking heavy oil into light oil and 21 chemical plants cracking light oil into petroleum. Video Explanation. 

Also there's a section:

**Moduled Cracking Ratio**
>Advanced Oil Processing
5
Heavy Oil Cracking
1
Light Oil Cracking
7

What do you want to do with those?

**Comment by deniszholob** [comment]
Yeah... gunna need to recalculate all of those
Also realized the table rates need updating too...
If you want to figure the calculations out feel free and create a new PR, ill merge this one though

**Comment by HonkingGoose** [comment]
I'm bad at calculating stuff, so the pull request for the table changes and recalculating can be done by somebody else. :smile:

**Pull request merged**